### DISCUSSION TITLE
Can't access API

### DISCUSSION THREAD

**Discussion by joshamaju** [realworld-apps/realworld#1557] [Community]
I can't access the API - https://api.realworld.io/api

in code

```json
{ "error": "Origin header is required" }
```

in browser

<img width="1552" alt="Screenshot 2024-08-16 at 23 15 52" src="https://github.com/user-attachments/assets/735a1fdc-cc3d-45d3-84fb-5e028b56dd78">

**Comment by geromegrignon** [comment]
Hello @joshamaju I added some restrictions upon the API usage right now as the server cost became bigger than the sponsoring help I get. You won't get any issue using the API from a frontend application.

**Reply by joshamaju** [reply]
Oh, sorry about that

**Reply by geromegrignon** [reply]
@joshamaju no problem, you were right to worry about that :)

**Comment by sumitsah** [comment]
![Screenshot 2024-12-02 230514](https://github.com/user-attachments/assets/703d8a76-21e1-4bef-b0b3-629d6853dab5)

App does not work in any environment as it used to work earlier. It is CORS issue.
